src/scala/org/pantsbuild/zinc/bootstrapper/Main.scala

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2017 Pants project contributors (see CONTRIBUTORS.md).
+ * Licensed under the Apache License, Version 2.0 (see LICENSE).
+ */
+
+package org.pantsbuild.zinc.bootstrapper
+
+import org.pantsbuild.zinc.scalautil.ScalaUtils
+import java.io.File
+import sbt.internal.util.ConsoleLogger
+
+object Main {
+  /*
+   Assume:
```

Comment:
May be worth a quick:
```scala
if (args.len != 7) {
  System.err.println("Usage: zinc-bootstrapper outputPath compilerInterface...");
  System.exit(1);
}
```

And also fleshing out these comments to talk about _where_ these files would come from (maybe links to their pages on maven central, or something)


Alternatively, scopt or scallop both look pretty lightweight, if we wanted to pull in one of those :)